config/tiling/README.md

```diff
@@ -53,3 +53,8 @@ particular patch, it is recommended to open the .geotiff version of ground truth
 If True, the tiling script uses Python's multiprocessing capability to process each AOI in parallel. This greatly 
 accelerates the tiling process. For testing or debugging purposes or for small dataset, we'd recommend keeping the 
 default `multiprocessing == False`.
+
+## Write destination multiband raster
+
+If True, will write the destination multiband raster from a VRT. Only affects cases when the source raster is composed 
+of singleband files being combined or if using a subset of bands from a multiband raster.
```

Comment:
Have two questions:
1. Current tests show that the destination multiband raster is written to the ./data folder anyways, even when the source raster is initially a multiband raster (not composed of several single-band rasters or not a subset of bands of a multiband raster) and I am personally OK with that. 
2. Do we want to write this dst multiband raster inside the associated project folder? currently it is in the ./data folder, as mentioned.

Reply:
1. solved. If a source raster is fine as is, I don't see any need to create a VRT. Thanks for pointing that out. Implementation now follows what the documentation says.
2.  Currently, this folder is set in tiling_segmentation but the "raw_data_dir" parameter. This could be changed with the "experiment_name" directory. However, it seems more logical to me that an enhance copy of a source raster should belong to a directory where the "raw data" is, rather than a folder specific to the current experiment. There may not be a consensus about this in the team and I'm certainly open to discussing this. In any case, a user wanting the enhanced version to be saved within the experiment folder could simply point the "raw data dir" to the same folder as the "experiment name", rather than ./data.
Let me know if this all makes sense.